lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2878,16 +3134,19 @@ FIRCircuitParser::parsePortList(SmallVectorImpl<PortInfo> &resultPorts,
 ///
 /// module ::= 'module' id ':' info? INDENT pohwist simple_stmt_block DEDENT
 /// module ::=
-///        'extmodule' id ':' info? INDENT pohwist defname? parameter* DEDENT
+///        'extmodule' id ':' info? INDENT pohwist defname? parameter* ref*
+///        DEDENT
 /// module ::=
-///        'intmodule' id ':' info? INDENT pohwist intname parameter* DEDENT
+///        'intmodule' id ':' info? INDENT pohwist intname parameter* ref*
```

Comment:
The IR has `internalPaths` on intrinsics and specification does not say they're supported one way or another, so decided to support parsing them for now like with extmodule's, so can resolve their uses if they occur.  If we have a use-case for it, we may not necessarily require the ref statement for all references on an intrinsic (like is required with extmodule; maybe an intrinsic generates a reference?), but for now handle similarly to extmodule and reject if not present.

Reply:
probably?